upload/src/addons/SV/RedisCache/_output/templates/admin/SV_Redis_info_Wrapper.html

```diff
@@ -0,0 +1,7 @@
+<xf:title>{{ phrase('redis_info') }}</xf:title>
+
+<div class="splash">
```

Comment:
This wrapper class was used in XF2 to make the overlap popup work compared to the injected inline version.